scout/dashboard.md

```diff
@@ -41,12 +41,33 @@ The repository tag list shows all tags for a repository. You can search for spec
 
 Each entry in the list shows the following details:
 
+- A checkbox to mark the tag for comparison to one other.
+
+  > **Tip**
+  >
+  > Compare two tag images by selecting the checkboxes next to them and selecting the **Compare images** button at the top of the list.
+  {: .tip }
+
 - The tag version. Clicking the link for version opens [the image layer view](#image-layer-view).
 - The operating system and architecture of the image.
 - The vulnerabilities for the tag version.
 - The last push for the tag version.
 - The base image and version used by the repository and the vulnerabilities for
   that version.
+- The size of the tag image.
```

Comment:
As above I don't think "tag image" is a correct term. I suggest we either use "image tag", "image", "tagged image" or "tag".

Reply:
```suggestion
- The size of the image tag.
```